sources/models/src/vmware-dev/override-defaults.toml

```diff
@@ -0,0 +1,12 @@
+[configuration-files.containerd-config-toml]
+# No override to path
+template-path = "/usr/share/templates/containerd-config-toml_aws-dev"
+
+# Docker
+[services.docker]
+restart-commands = ["/bin/systemctl try-restart docker.service"]
+configuration-files = ["proxy-env"]
+
+# Network
+[metadata.settings.network]
+affected-services = ["containerd", "docker", "host-containerd"]
```

Comment:
Should we also override the default host containers, since they aren't expected to work?